apps/api/src/app/controllers/__tests__/__snapshots__/resourceDescriptions.e2e.spec.ts.snap

```diff
@@ -4,38 +4,43 @@ exports[`GET /resources should return the expected result 1`] = `
 Array [
   Object {
     "description": "A term is a word, phrase, or sentence.",
-    "link": "/api/resources/terms",
+    "link": "/testApiPrefix/resources/terms",
```

Comment:
When we're in production will this be api instead of testApiPrefix?

Reply:
It will be whatever you want it to be based on your `prod.env` **or** your environment variables on the machine.